selene_sdk/utils/config_utils.py

```diff
@@ -218,7 +219,7 @@ def create_data_source(configs, output_dir=None, load_train_val=True, load_test=
             gen = torch.Generator()
             gen.manual_seed(configs["random_seed"])
             train_sampler = sampler_class(
-                train_dataset, replacement=False, generator=gen
+                train_dataset, replacement=False, generator=gen,
```

Comment:
Could you explain why do we need comma here?

Reply:
Generally speaking, we don't *need* this comma, but in Python you can leave trailing commas just for convenience (in case you want to add something later)

Sorry, I'm not sure why this line was changed in the first place, I'll amend this file, b/c this change is unnecessary